tasks/task03/word_stats.py

```diff
@@ -0,0 +1,22 @@
+import re
+
+texts = [
+   "Hello, World!",
+   "The world is mine",
+   "Hello, how are you?"
+]
+
+word_stats = {}
+line = 0
+
+for text in texts:
+    for word in re.sub('([^A-z^\s])+', '', text).lower().split():
```

Comment:
I think better to use this in function:
1. function to find all words in text and return splitting words
2. function to return stats with counting words
3. function to print stats

Functions are used to make code easier to test. You need to remember that